pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1351,4 +1351,11 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
 </div>
 </p></div>
 </p></div>
-</p></div>
\ No newline at end of file
+</p></div>
+<script>
+    $(document).ready(function() {
+       $('#step-collection-modal,#step-individual-modal,#step-query-modal,#step-post-modal,#step-relationship-modal,#step-function-modal').on('shown.bs.modal', function () {
```

Comment:
Do these modals have a common class name? If so, maybe use the class name instead of going through all id's? Or is there another reason you preferred to list all those ids?